tutorials/Working_with_fNIRS_data.py

```diff
@@ -0,0 +1,123 @@
+#
+# In the following example we have data from 20 channels plus one events
+# channel and we end up # with 20 HbO time series, 20 HbR time series and one
+# events time series. The provided data come from a dataset of neural
+# correlates of visual perception of biological motion in newborns.
+
+###############################################################################
+# Often fNIRS data contain time series for oxyhemoglobin (HbO) and
+# deoxyhemoglobin (HbR) concentrations and you want to process them both.
+# Using the *channel types* field in a :class:`mne.Info` class instance, you
+# can assign either 'hbo' or 'hbr' channel type to your channel.
+n_channels = 20  # number of physical channels
+sfreq = 15.625
+channel_names_fnirs = ['HbO %.2d' % i for i in range(1, n_channels + 1)] +
+                      ['HbR %.2d' % i for i in range(1, n_channels + 1)]
+channel_names = channel_names_fnirs + ['Events']
+channel_types = ['hbo'] * n_channels + ['hbr'] * n_channels + ['stim']
+info = mne.create_info(ch_names=channel_names, sfreq=sfreq,
+                       ch_types=channel_types, montage=None)
+
+###############################################################################
+# Import fNIRS data from numpy array using :class:`mne.io.RawArray`.
+raw_data = mne.io.RawArray(data=raw_ndarray, info=info, first_samp=0,
+                           verbose=None)
+
+###############################################################################
+# In our data array the last row contains event data, coded as different
+# numbers in a time series synchronous with the fNIRS recording. Let's find
+# such events using :func:`mne.find_events` and plot the events timeline using
+# :func:`mne.viz.plot_events`.
+events = mne.find_events(raw_data, stim_channel='Events', shortest_event=1)
+event_id = {'ISI': 1, 'Biological': 2, 'Rotation': 4, 'Scrambled': 8,
+            'Pause': 16}
+color = {1: 'black', 2: 'green', 4: 'magenta', 8: 'cyan', 16: 'yellow'}
+mne.viz.plot_events(events, raw_data.info['sfreq'], raw_data.first_samp,
+                    color=color, event_id=event_id)
+
+###############################################################################
+# Sometimes we want to filter out unwanted frequencies from time series. For
+# example, here we bandpass filter our data between 0.01 and 2 Hz.
+l_freq = 0.01  # high-pass filter cutoff ( __/¯¯¯ )
+h_freq = 2     # low-pass filter cutoff ( ¯¯¯\__ )
+raw_data.filter(l_freq, h_freq, fir_design='firwin')
+
+###############################################################################
+# In order to have a general glimpse of the dataset, we start by plotting all
+# channels into a single plot window. We choose to color-code HbO, HbR and
+# events data using red, blue and black respectively. We also add color-coded
+# events positions, using the same colors previously used in
+# :func:`mne.viz.plot_events`.
+scalings = dict(hbo=10e-6, hbr=10e-6, stim=1)
```

Comment:
And assuming you're using what can be considered a standard processing setup with standard signals, let's fix the scalings in the same file. We can add it to `whats_new.rst` as a bugfix.